FrontEndReact/src/View/Login/SetNewPassword.js

```diff
@@ -3,87 +3,277 @@ import 'bootstrap/dist/css/bootstrap.css';
 import ErrorMessage from '../Error/ErrorMessage.js';
 import { genericResourcePUT, validPasword } from '../../utility.js';
 import Login from './Login.js';
+import { Button, TextField, FormControl, Box, Typography } from '@mui/material';
+import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline';
+import CheckIcon from '@mui/icons-material/Check';
+
+
 
 class SetNewPassword extends Component {
     constructor(props) {
         super(props);
+
         this.state = {
             errorMessage: null,
-            passSet: false
+            passSet: false,
+            password: '',
+            password2: '',
```

Comment:
I suggest renaming `password2` -> `confirmationPassword` or something similar.

Reply:
I committed changes to changed password2 to confirmationPassword!